projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -80,9 +80,9 @@ export function useSetTier() {
 
 function TierSelector() {
 	// TODO: figure out how to handle different currencies
-	const products = useSelect( select => select( membershipProductsStore ).getProducts() )
-		.filter( product => product.subscribe_as_site_subscriber && product.interval === '1 month' )
-		.sort( ( p1, p2 ) => Number( p2.price ) - Number( p1.price ) );
```

Comment:
I don't see equivalent logic anywhere.  Do you limit to just the months to determine the label or is there a different mechanism I'm missing?

Reply:
You're right.. Nice catch!